src/main/windows/callsWidgetWindow.ts

```diff
@@ -209,7 +220,12 @@ export default class CallsWidgetWindow extends EventEmitter {
         this.setBounds(initialBounds);
     }
 
-    private onPopOutOpen = () => {
+    private onPopOutOpen = ({url}: {url: string}) => {
+        if (!urlUtils.isCallsPopOutURL(this.mainView.serverInfo.server.url, url, this.config.callID)) {
+            log.warn(`CallsWidgetWindow.onPopOutOpen: prevented window open to ${url}`);
+            return {action: 'deny' as const};
+        }
+
         return {
             action: 'allow' as const,
             overrideBrowserWindowOptions: {
```

Comment:
Can we swap the order? Deny by default and allow only when `urlUtils.isCallsPopOutURL` is `true` so that we follow secure by default principles.

```suggestion
    private onPopOutOpen = ({url}: {url: string}) => {
        if (urlUtils.isCallsPopOutURL(this.mainView.serverInfo.server.url, url, this.config.callID)) {
            return {
                action: 'allow' as const,
                overrideBrowserWindowOptions: {
                    autoHideMenuBar: true,
                },
            }
        }
        
        log.warn(`CallsWidgetWindow.onPopOutOpen: prevented window open to ${url}`);
        return {action: 'deny' as const};
```

Reply:
Sure. It actually simplifies the logic as we avoid the negation.